client/src/components/form/UploadAppFormStepper.jsx

```diff
@@ -66,6 +66,13 @@ const validate = values => {
     ) {
         errors.version.minVer = 'Cannot be higher than maximum version'
         errors.version.maxVer = 'Cannot be lower than minimum version'
+    } else if (values.version) {
+        if (!values.version.minVer) {
```

Comment:
Any reason not to include these fields in `requiredFields` above?

Reply:
Not really, it was already in a required-array but looking again I see they were originally in the required array for varChar fields which they arn't